.github/workflows/cabal.yml

```diff
@@ -29,32 +29,41 @@ jobs:
     - name: 📥 Checkout repository
       uses: actions/checkout@v1
 
+    - name: 🧰 Select optimal cabal version
+      run: |
+        case "$OS" in
+          Windows_NT)   echo "CABAL_VERSION=3.4.0.0-rc5"  >> $GITHUB_ENV;;
```

Comment:
Since when does rc5 exist?

Reply:
I think it might be an earlier rc actually, but that how the package is named in chocolatey because chocolatey packages are immutable and the rc3 published earlier was broken.  See https://github.com/Mistuke/CabalChoco/issues/2